src/stacked_borrows.rs

```diff
@@ -378,6 +382,7 @@ impl<'tcx> Stack {
         tag: SbTag,
         (alloc_id, alloc_range, offset): (AllocId, AllocRange, Size), // just for debug printing and error messages
```

Comment:
Same comment here about a doc comment explaining the `Size`

Reply:
I think this is already addressed in the comment?
```rust
    /// `range` refers the entire operation, and `offset` refers to the specific offset into the
    /// allocation that we are currently checking.
    ```

Reply:
Oh, this function signature didn't change, sorry.